bears/stylus/StylintBear.py

```diff
@@ -36,12 +42,236 @@ class StylintBear:
     SEE_MORE = 'https://github.com/SimenB/stylint'
 
     @staticmethod
+    def generate_config(filename, file,
+                        block_keyword: bool=None,
+                        brackets: bool=False,
+                        colons_for_property_declaration: bool=True,
+                        color_variables_for_hex_values: bool=True,
+                        spaces_after_commas: bool=True,
+                        spaces_after_comments: bool=True,
+                        allow_trailing_whitespace: bool=False,
+                        no_css_literals: bool=False,
+                        max_selector_depth: bool=False,
+                        check_duplicates: bool=True,
+                        efficient_properties: bool=True,
+                        extend_preference: str=None,
+                        indent_size: int=0,
+                        leading_zero: bool=None,
+                        max_errors: int=0,
+                        max_warnings: int=0,
+                        mixed_spaces_and_tabs: bool=False,
+                        variable_naming_convention: str=None,
+                        strict_naming_convention: bool=False,
+                        none_keyword: bool=False,
+                        check_no_important_keyword: bool=True,
+                        spaces_inside_parentheses: bool=None,
+                        placeholder: bool=True,
+                        prefix_vars_with_dollar: bool=True,
+                        semicolons: bool=False,
+                        sort_order: str='alphabetical',
+                        stacked_properties: bool=False,
```

Comment:
Yup, this is the default too, see https://github.com/SimenB/stylint#stackedproperties--default-never-never--false-